utils/webhook/github.handler.ts

```diff
@@ -46,21 +46,38 @@ export async function githubWebhookHandler(
         }
     });
 
-    if (!sync?.LinearTeam || !sync?.GitHubRepo) {
-        // Commenter is not found, post as application
-        if (githubEvent === "issue_comment" && action === "created") {
-            await createAnonymousUserComment(
-                body as IssueCommentCreatedEvent,
-                repository,
-                sender
-            );
-            return;
-        }
-
+    if (
+        (!sync?.LinearTeam || !sync?.GitHubRepo) &&
+        !process.env.LINEAR_APPLICATION_ADMIN_KEY

```

Comment:
why this check here?

Reply:
Because if a self hosted user don't have `LINEAR_APPLICATION_ADMIN_KEY` in his environment it should fail as before, since there is no admin key to create an anonymous issue with.